docs/modules/ROOT/pages/type-definitions/index.adoc

```diff
@@ -12,7 +12,7 @@
 - xref::type-definitions/default-values.adoc[Default Values] - Learn about different ways of setting default values for particular fields.
 - xref::type-definitions/database-mapping.adoc[Database Mapping] - Learn how to map the GraphQL Schema fields to custom Neo4j node and relationship properties.
 - xref::type-definitions/indexes-and-constraints.adoc[Indexes and Constraints] - Learn how to use schema directives to add indexes and constraints to your Neo4j database.
-- xref::type-definitions/introspector.adoc[Infer GraphQL Type Definitions] - If you have a existing database, you can learn how to automatically generate the type definition file from that.
+- xref::introspector.adoc[Infer GraphQL Type Definitions] - If you have a existing database, you can learn how to automatically generate the type definition file from that.
```

Comment:
Not 100% sure which is the correct way :sweat_smile: 
```suggestion
- xref::introspector.adoc[Infer GraphQL Type Definitions] - If you have an existing database, you can learn how to automatically generate the type definition file from that.
```

Reply:
I actually don't think this entry should be in this list at all, this is really just meant to be used as an index of the pages within (guidelines from the docs team). I would instead a suggestion a `NOTE` block at the top or bottom of the page, something like "Looking for how to generate type definitions based off your existing Neo4j database? Check out the Introspector page."